internal/custom_actions/utils.lua

```diff
@@ -129,9 +129,10 @@ function utils.getMatchedTrack(search_name, forward)
       end
     else
       local _, current_name = reaper.GetTrackName(track, "")
-      local has_no_name = current_name:match("Track ([0-9]+)", 1)
```

Comment:
I think it would be better to just keep this line instead of needing to call `string.lower` twice

Reply:
Hi! Just to be clear, you mean reverting just that line to what it was before string.lower?

Reply:
```
 else
      local _, current_name = reaper.GetTrackName(track, "")
      local has_no_name = current_name:match("Track ([0-9]+)", 1)
      current_name = current_name:lower()
      tracks_searched = tracks_searched + 1
      if not has_no_name and current_name:match(search_name:lower()) then
        return track
      end
```

Reply:
here's what I meant

Reply:
Got it. That's what I thought. And that makes sense. No need to make it lowercase. I'll get that committed and push.